spyderlib/plugins/inspector.py

```diff
@@ -334,6 +333,10 @@ def run(self):
                                                math=self.math_option,
                                                img_path=self.img_path)
                     html_text = sphinxify(doc['docstring'], context)
+                    if doc['docstring'] == '':
+                        html_text += '<hr>'

```

Comment:
Please change this line to

    html_text += '<div class="hr"></div>'

to better style this hr